pkg/controller/pravega/constants.go

```diff
@@ -19,6 +19,8 @@ const (
 	ssSecretVolumeName     = "ss-secret"
 	tlsVolumeName          = "tls-secret"
 	tlsMountDir            = "/etc/secret-volume"
+	caBundleVolumeName     = "ca-bundle"
```

Comment:
Please add something to the key and value that indicates this value is for ecs.... something on the lines of:
```
ecsCABundleVolumeName = "ecs-ca-bundle"
````

Reply:
it's not really just for ECS though, it's for any outbound connections.  Right now it's for ECS but other long term storage options could also need SSL trust